ios/Classes/Bindings/RecordingServiceBinding.swift

```diff
@@ -8,6 +8,10 @@ private enum EventKeys: String, CaseIterable {
 
 class RecordingServiceBinding: Binding {
 
+    override func onInit() {
+        VoxeetSDK.shared.recording.delegate = self
```

Comment:
Should we call super here? It will probably have no effect in this case, but generally it can be a good practice.

Reply:
Yes, Let's add `super` here